dist/debian/python3/control.template

```diff
@@ -8,7 +8,7 @@ Standards-Version: 3.9.5
 Rules-Requires-Root: no
 
 Package: %{product}-python3
-Architecture: amd64
+Architecture: any
```

Comment:
Don't forget to patch scylla-python3.git too (and let's revive that patch to use it)

Reply:
okay